src/test/ui/consts/const_let_eq.rs

```diff
@@ -5,6 +5,7 @@ struct Bar<T> { x: T }
 struct W(u32);
 struct A { a: u32 }
 
+#[allow(redundant_semicolon)]
```

Comment:
`redundant_semicolon` is not a part of the `unused` group? It probably should be.

(Also, it doesn't follow the [lint naming conventions](https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#lints).)

Reply:
I'll add it to the group and rename the lint to `redundant_semicolons`.